suite-native/atoms/src/Sheet/BottomSheetFlashList.tsx

```diff
@@ -39,107 +38,103 @@ const sheetContentContainerStyle = prepareNativeStyle<{
     paddingHorizontal: utils.spacings.sp16,
 }));
 
-const sheetWithOverlayStyle = prepareNativeStyle(_ => ({
-    flex: 1,
-    justifyContent: 'flex-end',
+const handleStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.borderDashed,
 }));
 
+const WindowOverlay = ({ children }: { children: ReactNode }) => {
+    if (Platform.OS === 'ios') {
```

Comment:
Please add comment why is this platform specific handling needed.

Reply:
removed, it was not necessary